legend-engine-xt-serviceStore-executionPlan/src/main/java/org/finos/legend/engine/plan/execution/stores/service/plugin/ServiceStoreExecutionState.java

```diff
@@ -44,6 +45,11 @@ public StoreState getStoreState()
         return this.state;
     }
 
+    public ServiceExecutor getServiceExecutor()
```

Comment:
ServiceStoreExecutionState should contain credentialProvider directly or within RuntimeConetxt